diff --git a/node_modules/rock/dist/src/lib/plugins/remoteCache.js b/node_modules/rock/dist/src/lib/plugins/remoteCache.js
index 784f723..9adabb2 100644
--- a/node_modules/rock/dist/src/lib/plugins/remoteCache.js
+++ b/node_modules/rock/dist/src/lib/plugins/remoteCache.js
@@ -72,7 +72,9 @@ ${output
             break;
         }
         case 'download': {
+            console.log('TEST ROCK START DOWNLOAD', artifactName);
             const localArtifactPath = getLocalArtifactPath(artifactName);
+            console.log('TEST ROCK LOCAL ARTIFACT PATH', localArtifactPath);
             const response = await remoteBuildCache.download({ artifactName });
             const loader = spinner({ silent: isJsonOutput });
             loader.start(`Downloading cached build from ${color.bold(remoteBuildCache.name)}`);
@@ -95,22 +97,28 @@ ${output
             break;
         }
         case 'upload': {
+            console.log('TEST ROCK START UPLOAD', artifactName);
             const localArtifactPath = getLocalArtifactPath(artifactName);
+            console.log('TEST ROCK LOCAL ARTIFACT PATH', localArtifactPath, 'args.binaryPath:', args.binaryPath);
             const binaryPath = args.binaryPath ?? getLocalBinaryPath(localArtifactPath);
             if (!binaryPath) {
+                console.log('TEST ROCK NO BINARY FOUND', artifactName);
                 throw new RockError(`No binary found for "${artifactName}".`);
             }
+            console.log('TEST ROCK BINARY FOUND', binaryPath);
             const buffer = await getBinaryBuffer(binaryPath, artifactName, localArtifactPath, args);
             try {
                 let uploadedArtifact;
                 const appFileName = path.basename(binaryPath);
                 const appName = appFileName.replace(/\.[^/.]+$/, '');
+                console.log('TEST ROCK UPLOAD remoteBuildCache START', artifactName, 'args.adHoc:', args.adHoc);
                 const { name, url, getResponse } = await remoteBuildCache.upload({
                     artifactName,
                     uploadArtifactName: args.adHoc
                         ? `ad-hoc/${artifactName}/${appName}.ipa`
                         : undefined,
                 });
+                console.log('TEST ROCK UPLOADED ARTIFACT', name, url);
                 const uploadMessage = `${args.adHoc ? 'IPA, index.html and manifest.plist' : 'build'} to ${color.bold(remoteBuildCache.name)}`;
                 const loader = spinner({ silent: isJsonOutput });
                 loader.start(`Uploading ${uploadMessage}`);
@@ -203,14 +211,17 @@ async function getInfoPlistFromIpa(binaryPath) {
     };
 }
 async function getBinaryBuffer(binaryPath, artifactName, localArtifactPath, args) {
+    console.log('TEST ROCK GET BINARY BUFFER', binaryPath, artifactName, localArtifactPath, args);
     // For ad-hoc, we don't need to zip the binary, we just upload the IPA
     if (args.adHoc) {
+        console.log('TEST ROCK GET BINARY BUFFER AD-HOC', binaryPath);
         return fs.readFileSync(binaryPath);
     }
     const zip = new AdmZip();
     const isAppDirectory = binaryPath.endsWith('.app') && fs.statSync(binaryPath).isDirectory();
     const absoluteTarballPath = args.binaryPath ?? path.join(localArtifactPath, 'app.tar.gz');
     if (isAppDirectory) {
+        console.log('TEST ROCK GET BINARY BUFFER IS APP DIRECTORY', binaryPath);
         const appDirectoryName = path.basename(binaryPath);
         if (args.binaryPath && !fs.existsSync(absoluteTarballPath)) {
             throw new RockError(`No tarball found for "${artifactName}" in "${localArtifactPath}".`);
@@ -224,12 +235,14 @@ async function getBinaryBuffer(binaryPath, artifactName, localArtifactPath, args
         zip.addLocalFile(absoluteTarballPath);
     }
     else {
+        console.log('TEST ROCK GET BINARY BUFFER ELSE', binaryPath);
         zip.addLocalFile(binaryPath);
     }
     const buffer = zip.toBuffer();
     if (isAppDirectory) {
         fs.unlinkSync(absoluteTarballPath);
     }
+    console.log('TEST ROCK GET BINARY BUFFER RETURN');
     return buffer;
 }
 function validateArgs(args, action) {
