diff --git a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
index 1cc3c26..ac287db 100644
--- a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
+++ b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
@@ -1,5 +1,4 @@
-import { existsSync } from 'node:fs';
-import { readdir } from 'node:fs/promises';
+import { existsSync, readdirSync } from 'node:fs';
 import { logger, spawn } from '@rock-js/tools';
 import { getAdbPath } from './adb.js';
 export async function findOutputFile(androidProject, tasks, device) {
@@ -40,7 +39,7 @@ export async function getInstallOutputFileName(appName, variant, buildDirectory,
     console.log('TEST getInstallOutputFileName BUILD DIRECTORY EXISTS', existsSync(buildDirectory));
 
     // list all files in the build directory
-    const files = await readdir(buildDirectory);
+    const files = readdirSync(buildDirectory);
     console.log('TEST getInstallOutputFileName FILES', files);
 
     const availableCPUs = await getAvailableCPUs(device);
@@ -61,6 +60,22 @@ export async function getInstallOutputFileName(appName, variant, buildDirectory,
         console.log('TEST getInstallOutputFileName OUTPUT FILE EXISTS case2:', outputFile);
         return outputFile;
     }
+
+
+
+  // check if there is a file like -debug.apk (missing app name)
+  if (existsSync(buildDirectory)) {
+    console.log('TEST getInstallOutputFileName ADDITIONAL OUTPUT CASE');
+    const pattern = `-${variant}.${apkOrAab}`;
+    console.log('TEST getInstallOutputFileName ADDITIONAL OUTPUT PATTERN', pattern);
+    const files = readdirSync(buildDirectory);
+    console.log('TEST getInstallOutputFileName ADDITIONAL OUTPUT FILES', files);
+    const matchingFile = files.find((file) => file.endsWith(pattern));
+    if (matchingFile) {
+        console.log('TEST getInstallOutputFileName ADDITIONAL OUTPUT FILE EXISTS', matchingFile);
+      return matchingFile;
+    }
+  }
     console.log('TEST getInstallOutputFileName OUTPUT FILE EXISTS NOT FOUND');
     logger.debug('Could not find the output file:', {
         buildDirectory,
