diff --git a/node_modules/@rock-js/platform-android/dist/src/lib/commands/buildAndroid/buildAndroid.js b/node_modules/@rock-js/platform-android/dist/src/lib/commands/buildAndroid/buildAndroid.js
index 873eeff..f0e1a25 100644
--- a/node_modules/@rock-js/platform-android/dist/src/lib/commands/buildAndroid/buildAndroid.js
+++ b/node_modules/@rock-js/platform-android/dist/src/lib/commands/buildAndroid/buildAndroid.js
@@ -21,10 +21,16 @@ export async function buildAndroid(androidProject, args, projectRoot, remoteCach
         fingerprintOptions,
         sourceDir: androidProject.sourceDir,
     });
+    console.log('TEST buildAndroid BINARY PATH', binaryPath);
     if (!binaryPath) {
+        console.log('TEST buildAndroid BINARY PATH NOT FOUND, RUNNING GRADLE');
         await runGradle({ tasks, androidProject, args, artifactName });
+        console.log('TEST buildAndroid GRADLE COMPLETED');
     }
+
+    console.log('TEST buildAndroid OUTPUT FILE BINARY PATH', binaryPath);
     const outputFilePath = binaryPath ?? (await findOutputFile(androidProject, tasks));
+    console.log('TEST buildAndroid OUTPUT FILE PATH', outputFilePath);
     if (outputFilePath) {
         logger.log(`Build available at: ${colorLink(relativeToCwd(outputFilePath))}`);
     }
diff --git a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
index 2938c29..4d555b1 100644
--- a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
+++ b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runAndroid/findOutputFile.js
@@ -3,41 +3,59 @@ import { logger, spawn } from '@rock-js/tools';
 import { getAdbPath } from './adb.js';
 export async function findOutputFile(androidProject, tasks, device) {
     const { appName, sourceDir } = androidProject;
+    console.log('TEST findOutputFile ANDROID PROJECT', androidProject);
+    console.log('TEST findOutputFile TASKS', tasks);
     const selectedTask = tasks.find((t) => t.startsWith('install') ||
         t.startsWith('assemble') ||
         t.startsWith('bundle'));
     if (!selectedTask) {
         return false;
     }
+    console.log('TEST findOutputFile SELECTED TASK', selectedTask);
     // handle if selected task includes build flavour as well, eg. installProductionDebug should create ['production','debug'] array
     const variantFromSelectedTask = selectedTask
         ?.replace('install', '')
         ?.replace('assemble', '')
         ?.replace('bundle', '')
         .split(/(?=[A-Z])/);
+    console.log('TEST findOutputFile VARIANT FROM SELECTED TASK', variantFromSelectedTask);
     // create path to output file, eg. `production/debug`
     const variantPath = variantFromSelectedTask?.join('/')?.toLowerCase();
+    console.log('TEST findOutputFile VARIANT PATH', variantPath);
     // create output file name, eg. `production-debug`
     const variantAppName = variantFromSelectedTask?.join('-')?.toLowerCase();
+    console.log('TEST findOutputFile VARIANT APP NAME', variantAppName);
     const apkOrBundle = selectedTask?.includes('bundle') ? 'bundle' : 'apk';
+    console.log('TEST findOutputFile APK OR BUNDLE', apkOrBundle);
     const buildDirectory = `${sourceDir}/${appName}/build/outputs/${apkOrBundle}/${variantPath}`;
+    console.log('TEST findOutputFile BUILD DIRECTORY', buildDirectory);
     const outputFile = await getInstallOutputFileName(appName, variantAppName, buildDirectory, apkOrBundle === 'apk' ? 'apk' : 'aab', device);
+    console.log('TEST findOutputFile OUTPUT FILE', outputFile);
+    console.log('TEST findOutputFile OUTPUT FILE PATH', outputFile ? `${buildDirectory}/${outputFile}` : undefined);
     return outputFile ? `${buildDirectory}/${outputFile}` : undefined;
 }
-async function getInstallOutputFileName(appName, variant, buildDirectory, apkOrAab, device) {
+export async function getInstallOutputFileName(appName, variant, buildDirectory, apkOrAab, device) {
+    console.log('TEST getInstallOutputFileName BUILD DIRECTORY', buildDirectory);
+    console.log('TEST getInstallOutputFileName BUILD DIRECTORY EXISTS', existsSync(buildDirectory));
+
     const availableCPUs = await getAvailableCPUs(device);
     // check if there is an apk file like app-armeabi-v7a-debug.apk
     for (const availableCPU of availableCPUs.concat('universal')) {
         const outputFile = `${appName}-${availableCPU}-${variant}.${apkOrAab}`;
+        //list all files in the build directory
+
         if (existsSync(`${buildDirectory}/${outputFile}`)) {
+            console.log('TEST getInstallOutputFileName OUTPUT FILE EXISTS case1:', outputFile);
             return outputFile;
         }
     }
     // check if there is a default file like app-debug.apk
     const outputFile = `${appName}-${variant}.${apkOrAab}`;
     if (existsSync(`${buildDirectory}/${outputFile}`)) {
+        console.log('TEST getInstallOutputFileName OUTPUT FILE EXISTS case2:', outputFile);
         return outputFile;
     }
+    console.log('TEST getInstallOutputFileName OUTPUT FILE EXISTS NOT FOUND');
     logger.debug('Could not find the output file:', {
         buildDirectory,
         outputFile,
diff --git a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runGradle.js b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runGradle.js
index 0ac455c..d9503d7 100644
--- a/node_modules/@rock-js/platform-android/dist/src/lib/commands/runGradle.js
+++ b/node_modules/@rock-js/platform-android/dist/src/lib/commands/runGradle.js
@@ -53,8 +53,11 @@ Tasks     ${color.bold(humanReadableTasks)}`);
         throw new RockError(hints ||
             'Failed to build the app. See the error above for details from Gradle.');
     }
+    console.log('TEST runGradle FIND OUTPUT FILE START',);
     const outputFilePath = await findOutputFile(androidProject, tasks);
+    console.log('TEST runGradle FIND OUTPUT FILE OUTPUT FILE PATH', outputFilePath);
     if (outputFilePath) {
+        console.log('TEST runGradle FIND OUTPUT FILE SAVE LOCAL BUILD CACHE', artifactName, outputFilePath);
         saveLocalBuildCache(artifactName, outputFilePath);
     }
 }
