diff --git a/node_modules/@rock-js/tools/dist/src/lib/build-cache/common.js b/node_modules/@rock-js/tools/dist/src/lib/build-cache/common.js
index d354835..c3a8598 100644
--- a/node_modules/@rock-js/tools/dist/src/lib/build-cache/common.js
+++ b/node_modules/@rock-js/tools/dist/src/lib/build-cache/common.js
@@ -39,9 +39,27 @@ export function getLocalArtifactPath(artifactName) {
     return path.join(getCacheRootPath(), BUILD_CACHE_DIR, artifactName);
 }
 export function getLocalBinaryPath(artifactPath) {
+    console.log('TEST ROCK TOOLS GET LOCAL BINARY PATH', artifactPath);
+//get part of path ending with ".rock"
+const pathBeforeRock = artifactPath.split('cache')[0].slice(0, -1);
+ console.log('TEST ROCK TOOLS GET ROCK PATH', pathBeforeRock);
+ //list structure from pathBeforeRock
+ const rockFiles = fs.readdirSync(pathBeforeRock);
+ console.log('TEST ROCK TOOLS GET FILES', rockFiles);
+ if(rockFiles.includes('cache')) {
+    const pathCache = path.join(pathBeforeRock, 'cache');
+    const rockFilesCache = fs.readdirSync(pathCache);
+    console.log('TEST ROCK TOOLS GET FILES CACHE', rockFilesCache);
+    if(rockFilesCache.includes('remote-build')) {
+        const pathRemoteBuild = path.join(pathCache, 'remote-build');
+        const rockFilesCacheRemoteBuild = fs.readdirSync(pathRemoteBuild);
+        console.log('TEST ROCK TOOLS GET FILES CACHE REMOTE BUILD', rockFilesCacheRemoteBuild);
+     }
+ }
     const files = fs.readdirSync(artifactPath);
     // Get the first non-hidden file as the binary
     const binaryName = files.find((file) => file && !file.startsWith('.'));
+    console.log('TEST ROCK TOOLS GET LOCAL BINARY NAME', binaryName);
     return binaryName ? path.join(artifactPath, binaryName) : null;
 }
 //# sourceMappingURL=common.js.map
\ No newline at end of file
